src/components/table/table.scss

```diff
@@ -39,3 +48,30 @@
   opacity: 1;
   transition: opacity 200ms 50ms ease-in;
 }
+
+.carbon-table__configure-link {
+  @include calc(height, "#{$tables__header-height} - 1px");
+  left: -25px;
+  position: absolute;
+  top: -1px;
+  width: 25px;
+
+  .carbon-link__anchor {
+    background-color: $grey-dark-blue-80;
+    border: 1px solid $border-color;
+    border-right: none;
+    border-radius: 25px 0 0 25px;
+    color: $white;
+    display: flex;
+    height: 100%;
+
+    &:hover {
+      background-color: $grey-dark-blue-90;
+      color: $white;
+    }
+
+    span {
```

Comment:
I expect this can be fully removed given the above comment, but for future reference using the `span` created by a component is risky as it might change, a `className` on the Carbon component (LInk in this case?) would be better